sql/moz-fx-data-shared-prod/telemetry_derived/newtab_merino_extract_v1/query.sql

```diff
@@ -0,0 +1,47 @@
+WITH deduplicated_pings AS (
+  SELECT
+    submission_timestamp,
+    document_id,
+    events,
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_live.newtab_v1`
+  WHERE
+    submission_timestamp > TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 DAY)
+  QUALIFY
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        DATE(submission_timestamp),
+        document_id
+      ORDER BY
+        submission_timestamp DESC
+    ) = 1
+),
+flattened_newtab_events AS (
+  SELECT
+    document_id,
+    submission_timestamp,
+    unnested_events.name AS event_name,
+    mozfun.map.get_key(
+      unnested_events.extra,
+      'scheduled_corpus_item_id'
+    ) AS scheduled_corpus_item_id,
+    mozfun.map.get_key(unnested_events.extra, 'position') AS position,
+    COUNT(1) OVER (PARTITION BY document_id, unnested_events.name) AS user_event_count
+  FROM
+    deduplicated_pings,
+    UNNEST(events) AS unnested_events
+    --filter to Pocket events
+  WHERE
+    unnested_events.category = 'pocket'
+    AND unnested_events.name IN ('impression', 'click', 'save', 'dismiss')
```

Comment:
Looks like we're only counting the `impression` and `click` events, should this drop `save` and `dismiss`?

Reply:
@fbertsch - yes, good catch, ty!